src/libraries/System.Data.Common/src/System/Data/SQLTypes/SqlXml.cs

```diff
@@ -128,7 +128,6 @@ internal static XmlReader CreateSqlXmlReader(Stream stream, bool closeInput = fa
 
         private static MethodInfo CreateSqlReaderMethodInfo
         {
-            [DynamicDependency("CreateSqlReader", typeof(System.Xml.XmlReader))]
```

Comment:
Is this change because the linker can now understand this use? I thought that reflecting on a method on a different assembly would still not always be detected.

Reply:
The bug you're referring to is this one: https://github.com/mono/linker/issues/943

It affects situation when linker analyzes a reflection pattern, finds out type X from assembly Y is needed, but assembly Y wasn't in the assembly closure linker initially started looking at (i.e. we would need to pull in a new assembly).

It can only happen in `Type.GetType("SomeType, SomeAssembly")` - the pattern in this file is safe from that bug because there's a typeof in the method body.